project/ws/app/src/lib/routes/app-toc/components/app-toc-home/app-toc-home.component.html

```diff
@@ -197,8 +150,83 @@
           </ng-container>
 
           <ng-container *ngIf="!skeletonLoader">
+            <ng-container *ngIf="content?.isInIntranet && showIntranetMsg">
+              <div class="flex flex-col gap-4" *ngIf="content?.completionStatus == 2 || certData">
+                <div class="pending-certificate" *ngIf="!certData">
+                  <div>Kindly be patient. The certificate generation normally takes up to 24 hours,
+                    however, it might take longer sometimes.</div>
+                  <div>If the issue persists, please mail us at mission.karmayogi@gov.in</div>
+                </div>
+                <ng-container *ngIf="certData">
+                  <a class="certificate_box" (click)="openCertificateDialog()">
+                    <img [src]="certData | pipeSafeSanitizer: 'url'" class="certificate_banner" />
+                    <div class="flex items-center certificate_overlay">
+                      <div class="flex flex-col text-center gap-1 p-4">
+                        <div class="font-bold text-white">Congratulations!</div>
+                        <div class="text-white">Your certificate of completion is ready for you to download.</div>
+                      </div>
+                    </div>
+                  </a>
+                </ng-container>
+              </div>
+
+              <div class="flex flex-row items-center gap-2 info-div" *ngIf="content?.isInIntranet && showIntranetMsg">
+                <mat-icon>info</mat-icon>
+                <ng-container i18n>This can be viewed only in the Intranet.</ng-container>
```

Comment:
hardcoded text should come from language files

Reply:
I haven't implemented the language yet! Once all the functionality is fixed. So I will fix language service